### PR TITLE
refactor: slightly simplify menu and dropdown logic

### PR DESCRIPTION
**Pull request by iambumblehead**

### CODE REVIEW

File: src/Menu.js [hunk 1]
```diff
@@ -55,24 +63,11 @@ const Menu = ({
...
-    <div className={className} aria-expanded={expanded.toString()}>
-      {display}
-    </div>
-  );
 };
 
 Menu.defaultProps = {
   ...BASE_DEFAULT_PROPS,
-  noOptionsDisplay: 'No options found',
-  expanded: false,
```
Comment (lwhiteley) [inline src/Menu.js]: will need to default to false if keeping this prop

Comment (lwhiteley) [reply src/Menu.js]: no worries, i see it was removed

**Comment by lwhiteley** [comment]
60fb70f  - commit message too long

File: src/Dropdown.js [hunk 2]
```diff
@@ -126,15 +126,17 @@ function Dropdown({
...
-      className={menuClass}
-      selected={selected}
-      options={options}
-      baseClassName={baseClassName}
-      noOptionsDisplay={noOptionsDisplay}
-      onSelect={(e, selectedValue, label) => setValue(selectedValue, label)}
-    />
+    <div className={className} aria-expanded>
+      <Menu
+        expanded
```
Comment (lwhiteley) [inline src/Dropdown.js]: remove the prop here as well

**Comment by lwhiteley** [comment]
my commit message style is conventional commits. you can read about it here if you want to https://www.conventionalcommits.org/en/v1.0.0/

**Comment by lwhiteley** [comment]
also remove `expanded` prop in `index.d.ts`

File: src/Dropdown.js (hunk 2 above)
Comment (iambumblehead) [reply src/Dropdown.js]: removed :)

**Comment by iambumblehead** [comment]
I removed the expanded value from the ts file... reading about the commit messages now

**Comment by lwhiteley** [comment]
> I removed the expanded value from the ts file... reading about the commit messages now

still one left

> also remove expanded prop in index.d.ts

**Comment by lwhiteley** [comment]
you can also in the repository settings, auto delete branches after merge

**Pull request merged**